.env.example

```diff
@@ -0,0 +1,21 @@
+AZURE_CLIENT_ID=
+AZURE_TENANT_ID=
```

Comment:
Can we give comments for each section explaining what they are and why a customer would need them?